packages/perseus/src/widgets/expression/expression.testdata.ts

```diff
@@ -4,11 +4,12 @@ import {
     type Version,
     type PerseusItem,
     type PerseusAnswerArea,
+    expressionLogic,
 } from "@khanacademy/perseus-core";
 
 const createItemJson = (
     widgetOptions: PerseusExpressionWidgetOptions,
-    version: Version,
+    widgetVersion: Version = expressionLogic.version as Version,
```

Comment:
Can we use `satisfies` instead of `as` here? Also, why do we need the cast at all? :)

Reply:
`expressionLogic` is of type `WidgetLogic` and not all widgets have versions so `version?: Version`

You're right thought, it was overkill. It works with `widgetVersion = expressionLogic.version`